orchagent/intfsorch.cpp

```diff
@@ -739,6 +796,21 @@ bool IntfsOrch::addRouterIntfs(sai_object_id_t vrf_id, Port &port)
     attr.value.u32 = port.m_mtu;
     attrs.push_back(attr);
 
+    if (isNatSupported)
+    {
+        attr.id = SAI_ROUTER_INTERFACE_ATTR_NAT_ZONE_ID;
+        if (m_nat_zone.find(port.m_alias) == m_nat_zone.end())
+        {
+            attr.value.u32 = DEFAULT_NAT_ZONE_ID;
```

Comment:
This is not required, its the SAI default

Reply:
Yes, removed it.